src/AzureAuth/Program.cs

```diff
@@ -44,12 +44,12 @@ private static void Main(string[] args)
                 backend = TelemetryOutput.ApplicationInsights;
             }
 
-            TelemetryConfig telemetryConfig = new TelemetryConfig()
-            {
-                Backend = backend,
-                IngestionToken = ingestionToken,
-                Async = true,
-            };
+            TelemetryConfig telemetryConfig = new TelemetryConfig(
+                eventNamespace: "azureauth",
+                backend: backend,
+                ingestionToken: ingestionToken,
+                useAsync: true,
+                envVarsToCollect: new[] { "SYSTEM_DEFINITIONID", "QBUILD_DISTRIBUTED" });

```

Comment:
Just curious if we can add these to Lasso? Because this seems like office specific terminology to me.

Reply:
Lasso gives us the functionality to add env vars in CLI that uses Lasso. Also, did not understand the latter comment. :)

Reply:
These are Microsoft specific, not Office specific. While it may be slightly less desirable to have them in someone else's telemetry (if they choose to set that up for themselves), we don't have a good alternative right now. Yes this could go into Lasso - but that would apply to every lasso app (which may or may not be desirable).